infrastructure/kube/keep-prd/bcoin-storageclass.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
+metadata:
+  name: bcoin
+  namespace: default
+provisioner: kubernetes.io/gce-pd
+parameters:
+  type: pd-ssd
+  replication-type: none
+reclaimPolicy: Retain
+allowVolumeExpansion: true
+mountOptions:
+  - debug
```

Comment:
Again for my own edification, what does debug do as a mount option?

Reply:
This should print debugging info on mount to help troubleshoot if things go sideways.